Tasks/CopyFilesOverSSHV0/sshhelper.ts

```diff
@@ -80,7 +80,7 @@ export class SshHelper {
                 this.sftpClient = null;
             }
         } catch(err) {
-            tl.debug('Failed to close SFTP client.');
```

Comment:
Nit: 
'Failed to close SFTP client: ' + err 
or
 `Failed to close SFTP client: ${err}`

Reply:
Sounds fair